app/assets/stylesheets/searchworks4/zero-results.css

```
@@ -0,0 +1,27 @@
+.zero-results {
+  #appliedParams .catalog_startOverLink {
+    display: none;
+  }
+
+  .clear-all {
+    --bs-primary-rgb: var(--stanford-digital-red-rgb)
+  }
+
+  .constraints-container {
+    display: block;
+  }
+
+  .applied-filter {
+    display: block;
+    margin-left: -0.25rem;
+
+    & .constraint-value, & .remove {
```

```suggestion
    .constraint-value,
    .remove {
```

Do we need the ampersands?